build/media_source/templates/site/cassiopeia/scss/blocks/_header.scss

```
@@ -83,6 +83,33 @@
       display: flex;
       flex: 1 1 0%;
       flex-direction: row;
+      :where(.mod-menu__sub:not(.mod-menu__sub *)) {
+        position: absolute;
+        top: 100%;
```

```suggestion
        top: 100%;
        z-index: 1001;
```

Thanks @exlemor  - can confirm.